cmd/buildd/main_unsupported.go

```diff
@@ -14,5 +14,5 @@ func appendFlags(f []cli.Flag) []cli.Flag {
 }
 
 func newController(c *cli.Context, root string) (*control.Controller, error) {
-	return nil, errors.New("invalid build")
+	return nil, errors.New("you need to build the daemon either with support for runc or containerd")
```

Comment:
Maybe even more specific, like "need to build daemon with either standalone or containerd build tag" ?

Reply:
sounds better, can do